src/SFA.DAS.ApprenticeCommitments.Api/nlog.config

```diff
@@ -10,7 +10,7 @@
 
   <targets>
     <target name="Console" layout="${simplelayout}" xsi:type="Console"/>
-    <target name="File" fileName="${basedir}/logs/${appName}.${shortdate}.log" layout="${simplelayout}" xsi:type="File"/>
+    <!--<target name="File" fileName="${basedir}/logs/${appName}.${shortdate}.log" layout="${simplelayout}" xsi:type="File"/>-->
```

Comment:
Is this meant to be comment out?  If so should we remove it?

Reply:
don't want to file log on the Server, but want it locally. Will do it the way we did it in the web, but in the meantime, I want to keep the reference so i can re-enable it locally